blockchain/state_manager/src/lib.rs

```diff
@@ -362,6 +361,7 @@ where
                     .unwrap(),
                 heights,
             )
+            .map_err(|e| anyhow::anyhow!("{}", e))
```

Comment:
`VM::new` should probably use `anyhow::Error`.

Reply:
Yes.